File: server/Game/Gate/GateHandler.cs
//////////////////////////////////////////////////////////////////////////
// 
// 文件：server/Game/Gate/GateHandler.cs
// 作者：Xoen Xie
// 时间：2023/06/19
// 描述：服务器相关定义
// 说明：
//
//////////////////////////////////////////////////////////////////////////
using GateA;
using XSF;

public class GateHandler : IGateHandler
{
    public void OnClientClose(uint nClientID)
    {
        Serilog.Log.Information("Client close, client id=" + nClientID);
        ActorManager.Instance.OnActorLogout(nClientID);
    }
}

File: server-cpp/source/game/server/src/GateHandler.cpp
//////////////////////////////////////////////////////////////////////////
//
// 文件：source/game/server/src/GateHandler.cpp
// 作者：Xoen Xie
// 时间：2020/02/29
// 描述：网关句柄
// 说明：
//
//////////////////////////////////////////////////////////////////////////
#include "GateHandler.h"
#include "XSF.h"
using namespace xsf;
#include "ActorManager.h"

void GateHandler::OnClientClose(uint32 nClientID)
{
    XSF_INFO("GateHandler::OnClientClose client id=%u", nClientID);
    ActorManager::Instance()->OnActorLogout(nClientID);
}